receiver.go

```diff
@@ -38,50 +38,54 @@ func (r *Receiver) DrainCredit(ctx context.Context) error {
 	return r.link.DrainCredit(ctx)
 }
 
-// Receive returns the next message from the sender.
+// Prefetched returns the next message that is stored in the Receiver's
+// prefetch cache. It does NOT wait for the remote sender to send messages
+// and returns immediately if the prefetch cache is empty.
 //
-// Blocks until a message is received, ctx completes, or an error occurs.
-// When using ModeSecond, you *must* take an action on the message by calling
-// one of the following: AcceptMessage, RejectMessage, ReleaseMessage, ModifyMessage.
-// When using ModeFirst, the message is spontaneously Accepted at reception.
-func (r *Receiver) Receive(ctx context.Context) (*Message, error) {
+// NOTE: Most callers will want to use `Receive`, which checks both the prefetch
```

Comment:
Should add the blurb about "you *must* take action" so it's clear callers still need to settle the message.

Reply:
Readded